controllers/routes.js

```diff
@@ -1,358 +1,244 @@
+            '/funding/funding-guidance/applying-for-funding'
+        ]
+    }),
+    contact: staticRoute({
+        path: '/contact',
+        template: 'pages/toplevel/contact',
+        lang: 'toplevel.contact',
+        aliases: [
+            '/about-big/contact-us',
+            '/help-and-support',
+            '/england/about-big/contact-us',
+            '/wales/about-big/contact-us',
+            '/scotland/about-big/contact-us',
+            '/northernireland/about-big/contact-us'
+        ]
+    }),
+    data: dynamicRoute({
+        path: '/data',
+        template: 'pages/toplevel/data',
+        lang: 'toplevel.data'
+    }),
+    jobs: staticRoute({
+        path: '/jobs',
+        template: 'pages/toplevel/jobs',
+        lang: 'toplevel.jobs',
+        aliases: ['/about-big/jobs', '/about-big/jobs/how-to-apply', '/about-big/jobs/current-vacancies']
+    }),
+    benefits: staticRoute({
+        path: '/jobs/benefits',
+        template: 'pages/toplevel/benefits',
+        lang: 'toplevel.benefits',
+        aliases: ['/about-big/jobs/benefits']
+    }),
+    under10k: staticRoute({
+        path: '/under10k',
+        template: 'pages/toplevel/under10k',
+        lang: 'toplevel.under10k',
+        aliases: ['/funding/Awards-For-All', '/funding/awards-for-all', '/awardsforall', '/a4a', '/A4A']
+    }),
+    over10k: staticRoute({
+        path: '/over10k',
+        template: 'pages/toplevel/over10k',
+        lang: 'toplevel.over10k'
+    }),
+    eyp: staticRoute({
+        path: '/empowering-young-people',
+        template: 'pages/toplevel/eyp',
+        lang: 'toplevel.eyp',
+        aliases: ['/global-content/programmes/northern-ireland/empowering-young-people']
+    }),
+    helpingWorkingFamilies: staticRoute({
+        path: '/helping-working-families',
+        template: 'pages/toplevel/working-families',
+        lang: 'toplevel.helpingWorkingFamilies',
+        aliases: ['/global-content/programmes/wales/helping-working-families']
+    })
 };
 
-// configure base paths for site sections
-const sectionPaths = {
-    toplevel: '',
-    funding: '/funding',
-    research: '/research',
-    about: '/about-big', // @TODO rename on launch
-    aboutLegacy: '/about-big' // used on the old site
+/**
+ * Funding Routes
+ */
+sections.funding.pages = {
+    root: dynamicRoute({
+        path: '/',
+        template: 'pages/toplevel/funding',
+        lang: 'toplevel.funding',
+        aliases: ['/home/funding']
+    }),
+    applyingForFunding: staticRoute({
+        path: '/funding-guidance/applying-for-funding/*'
+    }),
+    manageFunding: staticRoute({
+        path: '/funding-guidance/managing-your-funding',
+        template: 'pages/funding/guidance/managing-your-funding',
+        lang: 'funding.guidance.managing-your-funding',
+        aliases: ['/funding/funding-guidance/managing-your-funding/help-with-publicity', '/welcome', '/publicity']
+    }),
+
+    // @TODO: This is the only route that uses isWildcard??
```

Comment:
This is because it `POST`s to `/funding/funding-guidance/managing-your-funding/ordering-free-materials/$id` where `$id` is a product ID. It could quite easily `POST` to a generic endpoint and pass `$id` as data instead... future refactoring job maybe.